### PR TITLE
Add init, crop2raster and create_mask methods

### PR DESCRIPTION
**Pull request by adehecq**

### CODE REVIEW

File: GeoUtils/vector_tools.py
Comment (atedstone) [inline GeoUtils/vector_tools.py:53]: I don't think that the headline of this docstring accurately defines what the function actually does?

How about 

"Rasterize the vector features into a raster which has the extent of the provided raster file. 

Vector features which fall outside the bounds of the raster file are not written to the new mask file.

**Review by atedstone — COMMENTED** [review]
Nice! This is good to go, just pending slight docstring improvement before I give my seal of approval...

These couple of functions will simplify some of my recent work dramatically!

**Review by atedstone — APPROVED** [review]

**Review by iamdonovan — APPROVED** [review]

**Pull request merged**
